src/header/Header.scss

```diff
@@ -4,15 +4,22 @@ header {
   display: flex;
   align-items: center;
   justify-content: space-between;
-  margin-bottom: 1em;
+  margin-top: 1em;
```

Comment:
Thomas asked to change it to 0.75em

Reply:
ok..are you sure though ?

Reply:
I'm sure he asked, I have no idea how it looks like.

Reply:
I'm not quite sure about this ;). We were working together and he wanted to have top 1.0em and bottom 0.75em. WIll ask him and change if needed